curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff8f70dd32bc239b77a627.md

```diff
@@ -0,0 +1,164 @@
+---
+id: 63ff8f70dd32bc239b77a627
+title: Step 12
+challengeType: 0
+dashedName: step-12
+---
+
+# --description--
+
+Remember you're using template literals, so you have to add any of the items you destructured in the `forEach` with template interpolation.
+
+In JavaScript, we do template interpolation with the dollar sign (`$`) and curly braces:
+
+```js
+const name = "freeCodeCamp";
+const yob = 2014;
+
+//The interpolation
+const nameAndYob = `Info: ${name} was founded in ${yob}`;
+
+console.log(nameAndYob) // Info: freeCodeCamp was founded in 2014
+```
```

Comment:
While this is a good explanation, I don't think we need to add it here because campers will have had a lot of experience working with template literals in earlier projects